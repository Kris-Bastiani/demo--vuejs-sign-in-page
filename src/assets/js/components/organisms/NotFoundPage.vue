<template lang="pug">
	app-page
		.container.dsp__flex.pad__sml.w_max__reg
			h1 Page not found
</template>

<script>
import AppPage from '../molecules/AppPage.vue';

export default {
	components: { AppPage },
};
</script>
