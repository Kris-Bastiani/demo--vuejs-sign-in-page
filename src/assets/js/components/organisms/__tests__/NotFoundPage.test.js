import { shallowMount } from '@vue/test-utils';
import NotFoundPage from '../NotFoundPage.vue';

describe('NotFoundPage', () => {
	it('Matches snapshot', () => {
		const wrapper = shallowMount(NotFoundPage);
		expect(wrapper.element).toMatchSnapshot();
	});
});
