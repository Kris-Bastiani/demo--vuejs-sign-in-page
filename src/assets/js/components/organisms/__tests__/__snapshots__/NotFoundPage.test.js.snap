// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage Matches snapshot 1`] = `
<app-page-stub>
  <div
    class="container dsp__flex pad__sml w_max__reg"
  >
    <h1>
      Page not found
    </h1>
  </div>
</app-page-stub>
`;
